examples/kotlin/src/main/kotlin/com/example/booktest/postgresql/QueriesImpl.kt

```diff
@@ -275,31 +238,24 @@ class QueriesImpl(private val conn: Connection) : Queries {
                 results.getObject(7, OffsetDateTime::class.java),
                 (results.getArray(8).array as Array<String>).toList()
             )
-          if (results.next()) {
-              throw SQLException("expected one row in result set, but got many")
-          }
-          ret
-        }
+      if (results.next()) {
+          throw SQLException("expected one row in result set, but got many")
       }
+      ret
     }
   }
 
   @Throws(SQLException::class)
   override fun updateBook(
       title: String,
       tags: List<String>,
-      bookId: Int): ExecuteQuery {
-    return object : ExecuteQuery() {
-      override fun execute() {
-        conn.prepareStatement(updateBook).use { stmt ->
-          this.statement = stmt
-          stmt.setString(1, title)
+      bookId: Int) {
+    conn.prepareStatement(updateBook).use { stmt ->
+      stmt.setString(1, title)
           stmt.setArray(2, conn.createArrayOf("pg_catalog.varchar", tags.toTypedArray()))
```

Comment:
I assume this doesn't work on mysql? (Your other PR?)

Reply:
Nope. This field was changed to a comma delimitered varchar in the mysql version